### DISCUSSION TITLE
Access Version 0.4.0+ via npm

### DISCUSSION THREAD

**Discussion by alexdohm** [cdmoro/bootstrap-vue-3#784] [General]
version 0.4.3 has currently been released, but it's not accessible as a npm package. Is there a way to make this happen?

thanks!

**Comment by HenryUxDesugner** [comment]
When are we publishing new versions on NPM. the latest out there is 0.3.12

**Comment by HenryUxDesugner** [comment]
is the pipeline failing to publish?

**Reply by VividLemon** [reply]
https://github.com/cdmoro/bootstrap-vue-3/issues/771
